### DISCUSSION TITLE
Microsoft Edge隐藏New Bing图标 - 夏沫花火zzz🌙 (Muska_Ami)のLife

### DISCUSSION THREAD

**Discussion by Muska-Ami** [Muska-Ami/muska-ami-blog#59] [General]
https://blog.1l1.icu/2023/10/03/microsoft-edge-yin-cang-new-bing-tu-biao/ 

 Keep doing, keep loving❤.

**Comment by Muska-Ami** [comment]
要是没法打开那个edge的地址可以手动复制粘贴到地址栏打开

**Comment by shiftz300** [comment]
好像edge以前可以选择关闭侧边栏来着
